### ISSUE TITLE
Categories and Sub-categories

### ISSUE THREAD

**Issue by SunBreaker1** [Jays2Kings/tachiyomiJ2K#909]
### Why/User Benefit/User Problem
I have a lot of manga, webtoons, and the like save in the app. And when I say a lot, I mean an unhealthy amount.
Because of the obscene amount of mangas, webtoons and the like, I have a lot of categories created. 
Problem is with so many categories there comes a point where I have too many of them and its hard to stay organized.
That's where sub-categories come in.

### What/Requirements
The same way you're able to put folder into folder into folder, essentially unlimited times on Windows for example, I think that sub-categories would help organization.

ex. I have two categories "anime manga and anime favorites", with the ability to create sub-categories the two could merge into one main category with "anime manga and anime favorites" as the sub category options.

ex2. I have multiple categories dedicated just to creators and their works, the categories could become one big "Creators" category, and from there I could access all the creator's dedicated categories individually, saving a lot of space and improving visual clutter and organization.

I apologize if this is too wordy. If need be, I can provide a crude concept mock-up.

Thanks

**Comment by SrW0nka** [comment]
Similar to groups in chrome? if the case this can be extremely useful

**Comment by SunBreaker1** [comment]
> Similar to groups in chrome? if the case this can be extremely useful mainly in manga who have mor than one editon or as you mention 

It slipped my mind, but yes. Groups in Chrome is a great way to put it simply.

**Comment by Jays2Kings** [comment]
Duplicate of #735
